cosmic/src/evolv2.f

```diff
@@ -2696,8 +2723,14 @@ SUBROUTINE evolv2(kstar,mass,tb,ecc,z,tphysf,
 *
 * We have a giant whose envelope can absorb any transferred material.
 *
-            dm2 = dm1
+            dm2 = MIN(1.d0,t_acc_lim)*dm1
```

Comment:
check if ST allows full acc onto giants

Reply:
just confirming here - according to both Belczysnki+2008 and Dominik+2013, ST does not limit accretion onto the donor based on any thermal responses -- the accretion fraction is just applied as f_acc = 0.5. This is justified by a citation to Dewi & Tauris 2003 that says that massive stars accrete between 40%-70% of the donor's mass loss.